cluster/plan.go

```diff
@@ -437,6 +437,7 @@ func (c *Cluster) BuildKubeProxyProcess(host *hosts.Host, prefixPath string) v3.
 	}
 	Binds := []string{
 		fmt.Sprintf("%s:/etc/kubernetes:z", path.Join(prefixPath, "/etc/kubernetes")),
+		"/lib/modules:/lib/modules:ro",
```

Comment:
is it safe to mount the /lib/modules 
CC @moelsayed @yasker

Reply:
It should be safe (and it's RO) but I don't know the reason behind this. Why does this container need `/lib/modules/`? Is it going to load kernel module inside the container? If so, this not a good idea in general since it would change the host OS behavior as well.

Reply:
Yes, kube-proxy tries to load modules in the container if the modules are not yet loaded.

Search there for modprobe: https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/proxier.go

Reply:
I haven't dug deep into this. But what impact it would have on the host os?